### PR TITLE
Update to version 4

### PR DESCRIPTION
**Pull request by abhishekgupta5**
Support for twitter media creatives from account

### CODE REVIEW

**Comment by abhishekgupta5** [comment]
@jbabichjapan @tushdante Are you guys planning to update the gem for version 4 ? I see a few changes but version is still 3

File: lib/twitter-ads/account.rb [hunk 1]
```diff
@@ -94,6 +94,46 @@ def features
...
+
+    # Returns a collection of videos available to the current account.
+    #
+    # @param id [String] The Video ID value.
+    # @param opts [Hash] A Hash of extended options.
+    # @option opts [String] :video_ids Comma separated video ids
+    # @option opts [Boolean] :with_deleted Indicates if deleted items should be included.
+    # @option opts [String] :sort_by The object param to sort the API response by.
+    #
+    # @return A Cursor or object instance.
```
Comment (tushdante) [inline lib/twitter-ads/account.rb]: This endpoint has been deprecated in v4 (i.e., the `:account_id/videos` endpoint). Feel free to remove this code block entirely.

File: lib/twitter-ads/client.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 module TwitterAds
 
-  API_VERSION = '3'.freeze
+  API_VERSION = '4'.freeze
```
Comment (tushdante) [inline lib/twitter-ads/client.rb]: Lets keep this `3` for now, once all the other changes have been merged in, we'll make a separate version bump update.

**Review by tushdante — CHANGES_REQUESTED** [review]

**Comment by tushdante** [comment]
Thanks for the updates @abhishekgupta5 ! Appreciate all the help! Requested a few changes but otherwise LGTM! We should be able to merge this PR #180 and then make the change to v4.

**Pull request closed without merging**